odl_video/settings.py

```diff
@@ -539,3 +539,8 @@ def get_all_config_keys():
 MOIRA_CACHE_TIMEOUT = get_int('MOIRA_CACHE_TIMEOUT', 10800)
 HIJACK_ALLOW_GET_REQUESTS = True
 HIJACK_LOGOUT_REDIRECT_URL = '/admin/auth/user'
+
+EDX_ACCESS_TOKEN = get_string('EDX_ACCESS_TOKEN', None)
```

Comment:
Seems like the tokens generated automatically in edx have short lifespans, but the expiration can be manually changed.  I guess this is okay as long as we pick a token with a far-future expiration date?

Reply:
Yeah, that's what I was thinking. If a token expires, it will be easy to manually update this setting